opensrs/opensrsapi.py

```diff
@@ -473,19 +466,37 @@ def get_domains_by_expiredate(self, start_date, end_date, page=None):
         if qpage is None:
             qpage = 1
         while qpage is not None:
-            rsp = self._get_domains_by_expiredate(start_date, end_date, qpage)
-            data = rsp.get_data()
-            for domain in data['attributes']['exp_domains']:
+            data = self._get_domains_by_expiredate(start_date, end_date, qpage)
+            for domain in data['exp_domains']:
                 domains.append({
                     'domain': domain['name'],
                     'domain_expiration': domain['expiredate'],
                 })
-            if data['attributes']['remainder'] == '0':
+            if data['remainder'] == '0':
                 qpage = None
```

Comment:
That was an old code, I didn't want to re-factor/change it much in the scope of this ticket (since it will be removed at all on some stage).  But if you think we should change it - I don't mind.